content/hi/docs/setup/best-practices/multiple-zones.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+
+
+
+title: कई क्षेत्रों में चलाना
+weight: 20
+content_type: concept
+---
+
+<!-- overview -->
+
+यह पृष्ठ कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करता है।
+
+<!-- body -->
+
+## पृष्ठभूमि
+
+कुबेरनेट्स को डिज़ाइन या गया है ताकि एक कुबेरनेट्स क्लस्टर कई विफलता क्षेत्रों में चल सके,
+आमतौर पर जहां ये क्षेत्र फिट होते हैं
+एक तार्किक समूहन जिसे _क्षेत्र(region)_ कहा जाता है। प्रमुख क्लाउड प्रदाता एक क्षेत्र को परिभाषित करते हैं
+विफलता क्षेत्रों के एक सेट के रूप में (जिसे _उपलब्धता क्षेत्र भी कहा जाता है_) जो
+सुविधाओं का एक सुसंगत सेट प्रदान करते हैं: एक क्षेत्र के भीतर, प्रत्येक क्षेत्र समान एपीआई और सेवाएं प्रदान करता है।
+
+विशिष्ट क्लाउड आर्किटेक्चर का उद्देश्य इस संभावना को कम करना है कि एक जोन कि विफलता दूसरे जोन में सेवाएं बाधित ना करे।
+
+## विमान नियंत्रण व्यवहार
+
+सभी [विमान नियंत्रण अवयव](/docs/concepts/overview/components/#विमान-नियंत्रण-अवयव)
+
+विनिमेय संसाधनों के पूल के रूप में चल रहे समर्थन, प्रति दोहराया गया
+अवयव।
+
+जब आप क्लस्टर नियंत्रण विमान परिनियोजित करते हैं, तो इसकी प्रतिकृतियां रखें
+एकाधिक विफलता क्षेत्रों में विमान घटकों को नियंत्रित करें। यदि उपलब्धता एक महत्वपूर्ण चिंता है तो, कम से कम तीन विफलता क्षेत्रों का चयन करें और दोहराएं
+प्रत्येक व्यक्तिगत नियंत्रण विमान घटक (एपीआई सर्वर, अनुसूचक, आदि,
+क्लस्टर नियंत्रक प्रबंधक) कम से कम तीन विफलता क्षेत्रों में।
+यदि आप क्लाउड कंट्रोलर मैनेजर चला रहे हैं तो आपको चाहिए कि 
+इसे आपके द्वारा चुने गए सभी विफलता क्षेत्रों में भी दोहराएं।
+
+{{< note >}}
+कुबेरनेट्स एपीआई सर्वर के लिए क्रॉस-ज़ोन लचीलापन प्रदान नहीं करता है
+समापन बिंदु। उपलब्धता में सुधार के लिए आप विभिन्न तकनीकों का उपयोग कर सकते हैं
+क्लस्टर एपीआई सर्वर, जिसमें डीएनएस राउंड-रॉबिन, एसआरवी रिकॉर्ड, या
+स्वास्थ्य जांच के साथ एक तृतीय-पक्ष लोड संतुलन समाधान।
+{{< /note >}}
+
+## नोड व्यवहार
+
+कुबेरनेट्स स्वचालित रूप से पॉड्स को फैलाता है
+कार्यभार संसाधन (जैसे {{< glossary_tooltip text="डिप्लॉयमेंट(Deployment)" term_id="deployment" >}} or {{< glossary_tooltip text="स्टेटफुलसेट(StatefulSet)" term_id="statefulset" >}})
+एक क्लस्टर में विभिन्न नोड्स में। यह प्रसार मदद करता है
+विफलताओं के प्रभाव को कम करना।
+
+जब नोड शुरू होता है, तो प्रत्येक नोड पर क्यूबलेट स्वचालित रूप से नोड ऑब्जेक्ट के लिए {{< glossary_tooltip text="labels" term_id="label" >}} जोर देता है जो कुबेरनेट्स एपीआई में उस विशिष्ट क्यूबलेट का प्रतिनिधित्व करता है।
```

Comment:
Would
```suggestion
जब नोड शुरू होता है, तो प्रत्येक नोड पर क्यूबलेट स्वचालित रूप से नोड ऑब्जेक्ट के लिए {{< glossary_tooltip text="labelस" term_id="label" >}} जोर देता है जो कुबेरनेट्स एपीआई में उस विशिष्ट क्यूबलेट का प्रतिनिधित्व करता है।
```
work?

Reply:
it will confuse the reader, either `लेबलस` or `labels` will be better in my opinion

Reply:
“लेबलस (labels)” perhaps?